logic/src/commonMain/kotlin/com/wire/kalium/logic/data/session/SessionRepository.kt

```diff
@@ -205,4 +213,16 @@ internal class SessionDataSource(
     override suspend fun cookieLabel(userId: UserId): Either<StorageFailure, String?> = wrapStorageNullableRequest {
         authTokenStorage.getToken(userId.toDao())?.cookieLabel
     }
+
+    override suspend fun isAccountReadOnly(userId: UserId): Either<StorageFailure, Boolean> = wrapStorageRequest {
```

Comment:
```suggestion
    override suspend fun isAccountReadOnly(userId: UserId): Either<StorageFailure, Boolean> = wrapNullableStorageRequest {
```

wrapStorageRequest will return StorgeError.DataNotFound

Reply:
done